tests/unit/orcid/test_orcid_converter.py

```diff
@@ -0,0 +1,221 @@
+                                                                          'contributor-orcid': '',
+                                                                          'credit-name': 'Name2, Full'}]},
+                                        'external-identifiers': {'work-external-identifier': [{'external-identifier-id': 'arXiv:0000.00000',
+                                                                                               'external-identifier-type': 'ARXIV'}]},
+                                        'short-description': 'Abstract',
+                                        'title': {'subtitle': 'Subtitle', 'title': 'Title3'},
+                                        'type': 'REPORT'}
+
+
+def test_arxiv_missing_convertion():
+    record = {
+        "$schema": "http://localhost:5000/schemas/records/hep.json",
+        "abstracts": [
+             {
+                 "source": "arXiv",
+                 "value": "Abstract"
+             }
+        ],
+        "authors": [
+            {
+                "affiliations": [
+                    {
+                        "value": "Affil Inst."
+                    }
+                ],
+                "curated_relation": False,
+                "full_name": "Name, Full",
+                "inspire_id": "INSPIRE-00000000",
+                "orcid": "0000-0000-0000"
+            },
+            {
+                "affiliations": [
+                    {
+                        "value": "Affil Inst."
+                    }
+                ],
+                "curated_relation": False,
+                "full_name": "Name2, Full",
+                "inspire_id": "INSPIRE-00000001"
+            }
+        ],
+        "collections": [
+            {
+                "primary": "report"
+            },
+            {
+                "primary": "arXiv"
+            },
+            {
+                "primary": "book"
+            },
+            {
+                "primary": "HEP"
+            }
+        ],
+        "control_number": "0000000",
+        "external_system_numbers": [
+            {
+                "institute": "INSPIRETeX",
+                "obsolete": False,
+                "value": "Kats:2015sss"
+            },
+            {
+                "institute": "arXiv",
+                "obsolete": False,
+                "value": "oai:arXiv.org:0000.00000"
+            }
+        ],
+        "public_notes": [
+            {
+                "source": "arXiv",
+                "value": "17 pages, 3 figures"
+            }
+        ],
+        "self": {
+            "$ref": "http://localhost:5000/api/literature/0000000"
+        },
+        "titles": [
+            {
+                "source": "arXiv",
+                "title": "Title",
+                "subtitle": "Subtitle"
+            },
+            {
+                "title": "Title2"
+            },
+            {
+                "source": "other",
+                "title": "Title3"
+            }
+        ]
+    }
```

Comment:
All these are things that can be put in fixtures, so that you don't have to hardcode such huge blobs into python code. @mihaibivol @jacquerie can show you in case :)